Sources/Structures/NetworkService/NetworkServiceConfiguration.swift

```diff
@@ -59,9 +60,10 @@ public struct NetworkServiceConfiguration {
         sessionConfiguration.timeoutIntervalForResource = timeoutInterval
         sessionConfiguration.httpAdditionalHeaders = additionalHttpHeaders
 
-        serverTrustPolicies = [baseUrl: .disableEvaluation]
+        var updatedPolicies: [String: ServerTrustPolicy] = [:]
+        trustPolicies.forEach { updatedPolicies[$0.key.asHost] = $0.value }
```

Comment:
What does if two keys will have same hosts?
May be better solution to use
let updatePolicies = Dictionary(uniqueKeysValues: trustPolicies.map { ($0.key.asHost, $0.value) } which asserts on duplicate keys.